homeassistant/components/netatmo/climate.py

```diff
@@ -445,6 +465,16 @@ def update(self):
                                 self.boilerstatus
                                 and roomstatus["heating_status"]
                             )
+                        batterylevel = (
+                            self.homestatus.valves[roomstatus["module_id"]]
+                            .get("battery_level"))
+
+                    if batterylevel:
+                        if (roomstatus.get("battery_level") and
+                                batterylevel < roomstatus["battery_level"]):
+                            roomstatus["battery_level"] = batterylevel
+                        else:
+                            roomstatus["battery_level"] = batterylevel
```

Comment:
Both cases have the same result?

Reply:
Now I remember why the old PR was still WIP. You are 100% correct, that was not correct. Now the battery level is only updated if either there is no battery level set yet or the new battery level (reported by another module in the same room) is lower. This isn't ideal as multiple modules are grouped into one room which is the only controllable unit, so we can only have one battery level per climate entity (or can we have more?). Thanks once more for your extra critical pair of eyes.